snapshotter/config/config.go

```diff
@@ -54,6 +55,11 @@ type resolver struct {
 	Address string `toml:"address"`
 }
 
+type cache struct {
+	EvictOnConnectionFailure bool `toml:"evict_on_connection_failure" default:"true"`
+	PollConnectionFrequency  int  `toml:"poll_connection_frequency" default:"300"`
```

Comment:
could we rename to `PollConnectionFrequencySeconds` or similar to indicate unit?